crates/pevm/tests/common/mod.rs

```diff
@@ -32,19 +30,22 @@ pub fn for_each_block_from_disk(mut handler: impl FnMut(Block, InMemoryStorage<'
     let data_dir = std::path::PathBuf::from("../../data");
 
     // TODO: Deduplicate logic with [bin/fetch.rs] when there is more usage
-    let bytecodes: Bytecodes = bincode::deserialize_from(GzDecoder::new(BufReader::new(
+    let bytecodes = bincode::deserialize_from(GzDecoder::new(BufReader::new(
         File::open(data_dir.join("bytecodes.bincode.gz")).unwrap(),
     )))
     .unwrap();
 
+    let block_hashes = bincode::deserialize_from::<_, BlockHashes>(BufReader::new(
+        File::open(data_dir.join("block_hashes.bincode")).unwrap(),
+    ))
+    .unwrap();
+
     for block_path in fs::read_dir(data_dir.join("blocks")).unwrap() {
         let block_path = block_path.unwrap().path();
-        let block_number = block_path.file_name().unwrap().to_str().unwrap();
-
-        let block_dir = data_dir.join("blocks").join(block_number);
+        let block_dir = data_dir.join("blocks").join(block_path);
```

Comment:
Not sure but I guess `block_dir` is equals to `block_path`. If this is true, we can use `block_path` directly.

Reply:
Oh, you right! Totally missed it